core-web/libs/edit-content/src/lib/fields/dot-edit-content-file-field/components/dot-select-existing-file/components/dot-sidebar/dot-sidebar.component.scss

```diff
@@ -1,4 +1,11 @@
 :host {
     height: 100%;
     display: flex;
+    ::ng-deep {
+        .p-treenode-label {
```

Comment:
use the class `truncate-text` in the template ?

Reply:
It is not possible because this component is wrapped inside a PrimeNG component that handles the label.